src/main/kotlin/no/nav/familie/ks/sak/common/tidslinje/Periode.kt

```diff
@@ -14,6 +16,5 @@ data class Periode<T>(
 fun <T> List<Periode<T>>.tilTidslinje(): Tidslinje<T> = this.map { it.tilTidslinjePeriodeMedDato() }
     .sortedBy { it.fom }.tilTidslinje()
 
-fun <T> List<Periode<T>>.filtrerIkkeNull() = this.filter { it.verdi != null }
-
-data class IkkeNullbarPeriode<T>(val verdi: T, val fom: LocalDate, val tom: LocalDate)
+fun <T> List<Periode<T>>.filtrerIkkeNull(): List<Periode<T & Any>> =
```

Comment:
T & Any er kotlin sin måte å skrive @NotNull

Se "Support for definitely non-nullable types" [her](https://www.techiediaries.com/news/kotlin-1-7/)